docs/Installation-Guide.rst

```diff
@@ -191,12 +189,12 @@ Build Threadless Version (not Recommended)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The default build version of LightGBM is based on OpenMP.
-However, you can build the LightGBM without OpenMP support, but it is **strongly not recommended**.
+You can build the LightGBM without OpenMP support but it is **strongly not recommended**.
```

Comment:
```suggestion
You can build LightGBM without OpenMP support but it is **strongly not recommended**.
```

I know this was there before, but while you're touching this line we might as well clean up this "the LightGBM"